hearthstone/simulator/core/card_pool.py

```diff
@@ -2624,6 +2624,27 @@ def handle_event_powers(self, event: CardEvent, context: Union[BuyPhaseContext,
                         context.owner.play_blood_gem(card)
 
 
+class Shudderling(MonsterCard):
+    not_in_pool = True
+    tier = 1
+    monster_type = MONSTER_TYPES.NEUTRAL
+    base_attack = 1
+    base_health = 1
+
+    def base_battlecry(self, targets: List[MonsterCard], context: BuyPhaseContext):
+        battlecry_minion_queue = [card for card in context.owner.minions_played if card.battlecry and type(card) != type(self)]
+        while len(battlecry_minion_queue) > 0:
+            next_minion = battlecry_minion_queue.pop()
+            valid_targets = [card for card in context.owner.in_play if next_minion.valid_battlecry_target(card)]
+            chosen_targets = []
+            for _ in range(next_minion.num_battlecry_targets[-1]):  # TODO: how to deal with Defender of Argus???
```

Comment:
Hmm, is this the first time it matters where defender of argus is placed in a way that we can't just pretend it's a targeted battlecry?

Reply:
yes :(

Reply:
Hmm... easy solution is to just ignore it...